scripts/test/hyriseBenchmarkTPCC_test.py

```diff
@@ -1,23 +1,23 @@
 #!/usr/bin/env python3
 
-import sys
-
-from hyriseBenchmarkCore import close_benchmark, check_exit_status, initialize
+from compareBenchmarkScriptTest import CompareBenchmarkScriptTest
+from hyriseBenchmarkCore import close_benchmark, check_exit_status, initialize, run_benchmark
 
 
 def main():
+    build_dir = initialize()
+    compare_benchmarks_path = f"{build_dir}/../scripts/compare_benchmarks.py"
+    output_filename_1 = f"{build_dir}/tpcc_output_1.json"
 
     # Not explicitly setting all parameters and not testing all lines of the output. Many are tested in the TPCH test
-    # and we want to avoid duplication. First test single-threaded, then multi-threaded.
-
-    return_error = False
-
+    # and we want to avoid duplication. First test single-threaded, then multi-threaded, followed by a third run for
+    # compare_benchmark script tests.
```

Comment:
Where do I see the third run? I have only found output_filename_1 and output_filename_2.

Reply:
Not sure what you mean. There are three TPCC executions. But we only call the compare test once with the result of the last two runs.

So TPCC is executed three times, compare benchmark script once.

Reply:
It is just slightly (!) confusing to read about three tests but only seeing two filenames. A short sentence would suffice.

Reply:
```
    # Not explicitly setting all parameters and not testing all lines of the output. Many are tested in the TPCH test
    # and we want to avoid duplication. First TPC-C execution is single-threaded, second is multi-threaded. The third
    # execution is done to run the compare_benchmark script tests.
```

?